docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -1267,34 +1267,79 @@ Return low-level information on the image `name`
-                         "Tty": true,
-                         "OpenStdin": true,
-                         "StdinOnce": false,
-                         "Env": null,
-                         "Cmd": ["/bin/bash"],
-                         "Dns": null,
-                         "Image": "ubuntu",
-                         "Labels": {
-                             "com.example.vendor": "Acme",
-                             "com.example.license": "GPL",
-                             "com.example.version": "1.0"
-                         },
-                         "Volumes": null,
-                         "VolumesFrom": "",
-                         "WorkingDir": ""
-                 },
-         "Id": "b750fe79269d2ec9a3c593ef05b4332b1d1a02a62b4accb2c21d589ff2f5f2dc",
-         "Parent": "27cf784147099545",
-         "Size": 6824592
+      "ContainerConfig" : {
+        "MacAddress" : "",
+          "Domainname" : "",
+          "StdinOnce" : false,
+          "Entrypoint" : null,
+          "AttachStdin" : false,
+          "WorkingDir" : "",
+          "OnBuild" : null,
+          "AttachStdout" : false,
+          "NetworkDisabled" : false,
+          "OpenStdin" : false,
+          "PortSpecs" : null,
+          "Image" :
+            "37bea4ee0c816e3a3fa025f36127ef8ef0817b3f8fcd7b49eb7b26064f647bb0",
+          "Volumes" : null,
+          "Env" : null,
+          "User" : "",
+          "ExposedPorts" : null,
+          "Labels" : {},
+          "Cmd" : [
+            "/bin/sh",
+          "-c",
+          "#(nop) CMD [\"/bin/bash\"]"
+          ],
+          "Tty" : false,
+          "AttachStderr" : false,
+          "Hostname" : "9ec8c01a6a48"
+      },
+        "VirtualSize" : 188304295,
+        "Container" :
+          "a07d9edec0b3a777941a33087d8351d31f9aadce9418ad628de5e05dcebe8a3f",
+        "Parent" :
+          "37bea4ee0c816e3a3fa025f36127ef8ef0817b3f8fcd7b49eb7b26064f647bb0",
+        "Author" : "",
+        "Architecture" : "amd64",
+        "Config" : {
+          "Hostname" : "9ec8c01a6a48",
+          "AttachStderr" : false,
+          "Tty" : false,
+          "Cmd" : [
+            "/bin/bash"
+          ],
+          "ExposedPorts" : null,
+          "Labels" : {},
+          "User" : "",
+          "Env" : null,
+          "Volumes" : null,
+          "Image" :
+            "37bea4ee0c816e3a3fa025f36127ef8ef0817b3f8fcd7b49eb7b26064f647bb0",
+          "PortSpecs" : null,
+          "OpenStdin" : false,
+          "AttachStdout" : false,
+          "NetworkDisabled" : false,
+          "OnBuild" : null,
+          "WorkingDir" : "",
+          "AttachStdin" : false,
+          "Entrypoint" : null,
+          "StdinOnce" : false,
+          "Domainname" : "",
+          "MacAddress" : ""
+        },
+        "Comment" : "",
+        "Id" :
+          "07f8e8c5e66084bef8f848877857537ffe1c47edd01a93af27e7161672ad0e95",
+        "Os" : "linux",
+        "Tags" : [
+          "ubuntu:14.04",
+        "ubuntu:latest"
```

Comment:
pretty sure the spacing in this is section wrong given these tags don't line up. Check to make sure you don't have tabs, that usually messes it up.